verify/materialized_view_activity.sql

```diff
@@ -0,0 +1,7 @@
+-- Verify ggircs:materialized_view_activity on pg
+
+BEGIN;
+
+select * from ggircs_swrs.activity;
```

Comment:
we've started adding 'where false' to the select statements in verify so that the verify step doesn't return any data
```suggestion
select * from ggircs_swrs.activity where false;
```